docs/quickstart.md

```diff
@@ -0,0 +1,10 @@
+
+## Quickstart Guides
+Killercoda provide an interactive environment for exploring KubeVirt scenarios:
```

Comment:
@KiranSatyaRaj - almost there :) We just need some empty lines added between our intro sentences and our list items.
Markdown requires empty lines to separate formatting, otherwise it just keeps it as the same para, which you can see in the preview: https://deploy-preview-603--kubevirt-user-guide.netlify.app/quickstart/

Reply:
I'll fix that, where can I find this preview option for the changes like you did, can you please share it?

Reply:
Check out the tests your PR triggers below; one of those will be named `netlify/kubevirt-user-guide/deploy-preview` . It has a link. That's it.

Please also signoff your commits; right now it is failing the DCO check. You'll have to `git commit --amend -s` your commits and force push the PR again.

Reply:
+1 to what Miguel said.
You can also click on the 'Checks' tab at the top of the PR and see the netlify links.
I don't know how quickly it rebuilds after a commit but iirc it's pretty quick: https://deploy-preview-603--kubevirt-user-guide.netlify.app/quickstart/

Reply:
Thank you so much @aburdenthehand and @maiquebl, I've added the empty lines, please review it.

Reply:
@KiranSatyaRaj Perfect!
Before we can merge these you'll need to sign-off on all the additional commits:
https://github.com/kubevirt/user-guide/pull/603#issuecomment-1286710712

I think the easiest way to do this is an extension of what @maiqueb included above:
git commit --amend -s HEAD~4
And then force push the changes:
git push origin quickstartinToC -f

Reply:
I would actually squash all these commits into a single one, then amend it. 

Given what this change does, I don't see much motivation to have 5 different commits.